feature_tests/js/api/MyStruct.d.ts

```diff
@@ -1,5 +1,6 @@
 // generated by diplomat-tool
 import type { MyEnum } from "./MyEnum"
+import type { MyZst } from "./MyZst"
```

Comment:
obesrvation: I guess importing the error types don't hurt, especially if we want to reference them in documentation